quickwit/quickwit-query/src/query_ast/full_text_query.rs

```diff
@@ -130,10 +131,7 @@ impl FullTextParams {
                     .collect();
                 Ok(TantivyBoolQuery::build_clause(operator, leaf_queries).into())
             }
-            FullTextMode::BoolPrefix {
-                operator,
-                max_expansions,
```

Comment:
I'm a bit worried by the removal of `max_expansions`. It seems that the parameter exists in the [ES API](https://github.com/elastic/elasticsearch/blob/8.11/server/src/main/java/org/elasticsearch/index/query/MatchBoolPrefixQueryBuilder.java)

Reply:
that's interesting, it's not documented in https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-bool-prefix-query.html :confused:

Reply:
I think we should keep the `max_expansions` then.